deployment/charts/universalnode/templates/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app.kubernetes.io/app: {{ .Values.name }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/app: universalnode
```

Comment:
Please, replace `universalnode` here and on lines 14 and 17 with `{{ .Values.name }}`. Thanks